### DISCUSSION TITLE
Internal-Config配置增加取反操作

### DISCUSSION THREAD

**Discussion by Ctrl-Ling** [shuax/MouseInc.Settings#283] [Ideas]
是否可以支持对目前配置的True/False的取反操作？
场景：本人日常有使用大写锁定提示的功能，但是目前的Internal-Pause操作也并不会将锁定提示关闭，本人想在打游戏时关闭大写锁定提示，如果使用Internal-Config，则需要分别设置2个操作单独控制开和关

目前只能对Internal-Config配置单独设置开或者关状态，这意味着我需要设置两个快捷键进行开和关的控制，是否能增加对当前配置取反的一个功能，实现只通过一个快捷键即可控制开关，类似Internal-Pause这样？

**Comment by Ctrl-Ling** [comment]
可能说的不是很明确，比如配置一个["Internal", "Config", "CapsLockLed", "reverse"]动作
当CapsLockLed值为true时，触发此动作会将CapsLockLed值设为false；
当CapsLockLed值为false时，触发此动作会将CapsLockLed值设为true；

**Comment by shuax** [comment]
你说的有道理，但是我想在mouseinc3彻底解决类似问题
